data/json/items/resources/wood.json

```diff
@@ -72,13 +72,42 @@
     "category": "spare_parts",
     "melee_damage": { "bash": 18 }
   },
+  {
+    "type": "GENERIC",
+    "id": "soaking_long_pole",
+    "name": { "str_sp": "soaking long pole" },
+    "description": "A stout, ten-foot pole.  Could be used similarly to a spear.  It's currently soaked with oil and quite slippery.  Once the oil penetrates the wood it can be wiped down and used for further projects.",
+    "//": "a long pole is a single piece of wood, and the heavy stick just isn't large enough.",
+    "color": "brown",
+    "symbol": "/",
+    "material": [ "wood" ],
+    "techniques": [ "WBLOCK_1" ],
+    "weight": "2250 g",
+    "volume": "3750 ml",
+    "longest_side": "305 cm",
+    "flags": [ "POLEARM", "SPEAR", "REACH_ATTACK", "REACH3", "NONCONDUCTIVE", "ALWAYS_TWOHAND" ],
+    "weapon_category": [ "POLEARMS" ],
+    "price": "40 USD",
+    "to_hit": { "grip": "bad", "length": "long", "surface": "point", "balance": "good" },
+    "use_action": {
+      "target": "long_pole",
+      "msg": "You wipe down the long pole.",
+      "moves": 1000,
+      "type": "delayed_transform",
+      "transform_age": 15000,
+      "not_ready_msg": "The long pole is still soaking.",
+      "//": "4 hours"
+    },
+    "price_postapoc": "50 cent",
+    "qualities": [ [ "HAMMER", 1 ] ],
+    "melee_damage": { "bash": 14 }
+  },
   {
     "type": "GENERIC",
     "id": "long_pole",
-    "name": { "str": "long pole" },
+    "name": { "str_sp": "long pole" },
```

Comment:
Why this change? Plural of pole is poles, I think?

Reply:
I don't remember why that was changed. It should be just "str", since you can put an s at the end for the plural I think.

Reply:
Yes, it should be str.

Reply:
Yes, it should be str.